File: feedback-system-app-ui-master-0b382aa22285ca6b6c23fd4106525356895d5d2a/feedback-system-app-ui-master-0b382aa22285ca6b6c23fd4106525356895d5d2a/src/app/_models/userModel.ts
export class UserModel{
    associateId: number;
    associateName: string;
    email: string;
    roleId?: number;
    designation: string;
    buid?: number;
    addedUserId?: number;
    addedDate?: Date;
    isDeleted: boolean;
    deletedUserId?: number;
    deletedDate: string;
}

File: feedback-system-app-api-master-c0c8aa35f7bfaebb564ad7ea8d24da59bb7738e9/feedback-system-app-api-master-c0c8aa35f7bfaebb564ad7ea8d24da59bb7738e9/Feedback_System/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Feedback_System.Models
{
    public class UserModel
    {
        public int AssociateId { get; set; }
        public string AssociateName { get; set; }
        public string Email { get; set; }
        public int? RoleId { get; set; }
        public string Designation { get; set; }
        public int? Buid { get; set; }
        public int? AddedUserId { get; set; }
        public DateTime? AddedDate { get; set; }
        public bool IsDeleted { get; set; }
        public int? DeletedUserId { get; set; }
        public string DeletedDate { get; set; }
    }
}
